konfera/register/forms.py

```diff
@@ -3,13 +3,13 @@
 from konfera.models import Ticket
 
 
-class VolunteerRegistrationForm(forms.ModelForm):
+class RegistrationForm(forms.ModelForm):
     class Meta:
         model = Ticket
         fields = ('title', 'first_name', 'last_name', 'email', 'phone', 'description')
 
     def __init__(self, *args, **kwargs):
```

Comment:
Add a parameter that would decide if description field should be required of not. By default we can have description optional field.